src/FixerRunner/Application/FixerFileProcessor.php

```diff
@@ -44,6 +46,7 @@ public function __construct(
         array $fixers
     ) {
         $this->fixers = $this->sortFixers($fixers);
+        $this->isDebug = $easyCodingStandardStyle->isDebug();
```

Comment:
is this need to be applied in `EasyCodingStandardApplication` and `File` object as well?

Reply:
I did it only here to keep the PR minimal. `isDebug` is not per se slow/hungry, but in this special case which is called very frequently